README.md

```diff
@@ -132,6 +132,23 @@ External adapters are what make Chainlink easily extensible, providing simple in
 
 For more information on creating and using external adapters, please see our [external adapters page](https://docs.chain.link/docs/external-adapters).
 
+## Verify Official Chainlink Releases
+
+We use `cosign` with OIDC keyless signing during the [Build, Sign and Publish Chainlink](https://github.com/smartcontractkit/chainlink/actions/workflows/build-publish.yml) workflow.
+
+It is encourage for any node operator building from the official Chainlink docker image to verify the tagged release version was did indeed built from this workflow.
+
+You will need `cosign` in order to do this verification.
+
+`brew install cosign`
```

Comment:
Do we want to add different instructions for systems other than macOS here?

Reply:
good call

Reply:
Pointing to the installation instructions from their website should be good: https://docs.sigstore.dev/system_config/installation/

Reply:
yep did exactly that :)